hacspec/src/lib.rs

```diff
@@ -1447,9 +1502,17 @@ mod tests {
     use super::*;
     // test vectors (TV)
 
+    // message_1 (first_time)
+    const METHOD_TV_FIRST_TIME: u8 = 0x03;
```

Comment:
Mainly for main understanding: the test vectors are now all supposed to reflect an update to `draft-ietf-lake-traces-05` which will account for the `c_r` change?

Reply:
Yes, the test vectors reflect editor's copy of `draft-ietf-lake-traces` which is available at https://lake-wg.github.io/edhoc/draft-ietf-lake-traces.html

With this code, I confirmed the *second* trace in the draft. The draft will be published as -06 as soon as we receive the confirmation that the *first* trace (using X25519 curve) is correct.

Reply:
Thanks!